Libs/vtkSegmentationCore/vtkOrientedImageDataResample.cxx

```diff
@@ -1881,13 +1881,9 @@ void vtkOrientedImageDataResample::CastImageForValue(vtkOrientedImageData* image
       {
       scalarType = VTK_DOUBLE;
       }
-    else if (value > VTK_LONG_MAX || value < VTK_LONG_MIN)
-      {
-      scalarType = VTK_FLOAT;
-      }
     else if (value > VTK_INT_MAX || value < VTK_INT_MIN)
       {
-      scalarType = VTK_LONG;
+      scalarType = VTK_FLOAT;
```

Comment:
Can a `float` represent all `long int` values? If not then it may be better to use `double`.